src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -187,8 +188,8 @@ mod tests {
         .unwrap();
 
         let timestamp0 = Utc::now();
+        thread::sleep(sleep_time);
         let timestamp1 = Utc::now();
-        assert_ne!(timestamp0, timestamp1);
```

Comment:
Just one question before I approve this. As I recall from reading the metadata document, if both times are the same, one timestamp is incremented, correct? So is this why we don't need to assert that they're different?

Reply:
I have decided to trust the sleep() guarantee implicitly, that 1 second must elapse between calls. The assertion was there before because I had a hunch that the problem we're encountering now might sometime occur, that it wasn't absolutely guaranteed not to occur...and since the test wouldn't work w/out it, an assert was appropriate.